### PR TITLE
Issue-4790

### PR DESCRIPTION
**Pull request by habibrhmn**
### Describe your changes :

Worked on updating workflow to build and push Ingestion Dev Packages with approvals. This is for the issue 4790
- [x] Improvement

### CODE REVIEW

File: .github/workflows/py-ingestion-publish.yml [hunk 1]
```diff
@@ -41,10 +41,13 @@ jobs:
             twine check dist/*; \
             twine upload -r testpypi dist/*
   publish:
+    needs: [publish-test]
     runs-on: ubuntu-latest
     environment: release
     steps:
       - uses: actions/checkout@v2
+        with:
+          approvers: akash-jain-10, pmbrull, harshach
```
Comment (pmbrull) [inline .github/workflows/py-ingestion-publish.yml:50]: Hi @habibrhmn , is this needed? When I configured the environments I already set protection rules with some reviewers in there

Comment (habibrhmn) [reply .github/workflows/py-ingestion-publish.yml:50]: Can i get the variables. I will add them in the file. @pmbrull

Comment (pmbrull) [reply .github/workflows/py-ingestion-publish.yml:50]: But actually, the question is why do we need to add this in this workflow? It feels easier to just configure the environment directly and have the protection rules automatically running for any workflow that aims at `release`

Comment (pmbrull) [reply .github/workflows/py-ingestion-publish.yml:50]: hi @habibrhmn , oki np. Then we could just remove those. This is the list we have configured at the environment 
image

**Review by pmbrull — APPROVED** [review]

**Pull request merged**